core/src/main/java/org/infinispan/container/entries/PrimitiveEntrySizeCalculator.java

```diff
@@ -131,32 +132,27 @@ static int calculateMaxIntCache() {
    }
 
    /**
-    * Returns a sun.misc.Unsafe.  Suitable for use in a 3rd party package.
-    * Replace with a simple call to Unsafe.getUnsafe when integrating
-    * into a jdk.
+    * Returns a sun.misc.Unsafe.  Suitable for use in a 3rd party package. Replace with a simple call to
+    * Unsafe.getUnsafe when integrating into a jdk.
     *
     * @return a sun.misc.Unsafe
     */
    static Unsafe getUnsafe() {
```

Comment:
not related to the PR itself, but wouldn't make sense to use the "cached" unsafe in `UnsafeHolder`?

Reply:
We would need to change the visibility of the UnsafeHolder class/field